### DISCUSSION TITLE
Publish from script size limit?

### DISCUSSION THREAD

**Discussion by Jardland** [arendst/Tasmota#16099] [Q&A] [answered]
It seems that you only can publish 255 characters, is there any way to expand that. It works OK from console.

**Comment by gemu2015** [comment]
it is fixed by this define (you must specify double the size you want)

#define SCRIPT_CMDMEM 512

in this version you may define SCRIPT_CMDMEM in user_override

[xdrv_10_scripter.ino.zip](https://github.com/arendst/Tasmota/files/9200252/xdrv_10_scripter.ino.zip)

**Comment by gemu2015** [comment, marked as answer]
in pr#16106 i introduce a variable cbs (command buffer size)
where you can adjust max command size up to 4096 (default is 256)
